lib/matplotlib/backends/backend_qt5.py

```diff
@@ -233,20 +233,12 @@ def _update_figure_dpi(self):
     @_allow_super_init
     def __init__(self, figure):
         _create_qApp()
-        figure._original_dpi = figure.dpi
-
         super(FigureCanvasQT, self).__init__(figure=figure)
 
+        figure._original_dpi = figure.dpi
```

Comment:
Does the super chain no longer use `_original_dpi`?

Reply:
AFAICT the only places it is accessed are _update_figure_dpi (which is called later) and print_figure (which is irrelevant here).

Reply:
I suspect this got sorted out as part of the initial draw / re-entrant paint fix.